assets/facets.js

```diff
@@ -214,12 +244,18 @@ customElements.define('price-range', PriceRange);
 class FacetRemove extends HTMLElement {
   constructor() {
     super();
-    this.querySelector('a').addEventListener('click', (event) => {
-      event.preventDefault();
-      const form = this.closest('facet-filters-form') || document.querySelector('facet-filters-form');
-      form.onActiveFilterClick(event);
+    const facetLink = this.querySelector('a');
+    facetLink.addEventListener('click', this.closeFilter.bind(this));
+    facetLink.addEventListener('keydown', (event) => {
```

Comment:
WCAG [2.5.2 Pointer Cancellation](https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html) requires use of "up" events (`click()`, `keyup()`, `mouseup()`, `touchend()`) in order to allow users to "cancel" the event by swiping/mousing/<kbd>Esc</kbd> away from of the control.

```suggestion
    facetLink.addEventListener('keyup', (event) => {
```